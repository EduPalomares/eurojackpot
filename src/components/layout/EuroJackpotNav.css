nav {
	width: 100%;
	max-width: 960px;
	margin:0 auto;
	margin-top: 30px;
}

nav ul {
	padding: 0;
}

nav li {
	width: 33.32%;
	height: 42px;
	display: inline-block; 
	line-height:42px;
	text-align: center;
	background-color: #f9f9f9;
	background-image: linear-gradient(#fff 30%,#f9f9f9);
    box-shadow: inset 2px 2px 0 -1px #ccc, inset 0 -4px 0 -2px rgba(0,0,0,.2);
	cursor:pointer;
	overflow: hidden;
}

nav li:hover {
	background-color: #f2f2f2;
    background-image: none;
    color: #97bd00;
}

nav li:first-child {
    border-radius: 5px 0 0 5px;
    box-shadow: inset 1px 1px 0 0 #ccc, inset 0 -3px 0 -1px rgba(0,0,0,.2);
}

nav li:last-child {
    border-radius: 0 5px 5px 0;
    box-shadow: inset 0 0 0 1px #ccc, inset 0 -3px 0 -1px rgba(0,0,0,.2);
}

nav a.active li {
    background: #ffe066;
    box-shadow: inset 0 3px 0 -1px rgba(0,0,0,.2);
}

nav a {
	text-decoration: none;
	font-family: 'Arial'; 
	font-weight: bold;
	font-size: 12px;
	color: #97bd00;
}

nav a.active {
	color: #535353;
}

@media only screen and (max-width:960px) {
	nav {
		padding:0 1%;
	}
}

@media only screen and (max-width:360px) {
	nav {
		display:none;
	}
}