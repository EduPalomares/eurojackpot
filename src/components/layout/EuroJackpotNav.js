import React from 'react';
import { NavLink } from "react-router-dom";
import './EuroJackpotNav.css';

export default class EuroJackpotNav extends React.Component {

	render() {
		return (
			<nav>
				<ul>
					<NavLink exact to="/" activeClassName="active"><li>EuroJackpot</li></NavLink>
					<NavLink to="/results-winning-numbers" activeClassName="active"><li>EuroJackpot Results & Price Breakdown</li></NavLink>
					<NavLink to="/help" activeClassName="active"><li>EuroJackpot Help & Faq</li></NavLink>
				</ul>
			
			</nav>
		)
	}
}