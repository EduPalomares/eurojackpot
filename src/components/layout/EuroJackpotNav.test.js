import React from 'react';
import ReactDOM from 'react-dom';
import EuroJackpotNav from './EuroJackpotNav';

test('render match snapshot', () => {
    const wrapper = shallow(
		<EuroJackpotNav />
    );
    expect(wrapper).toMatchSnapshot();
});