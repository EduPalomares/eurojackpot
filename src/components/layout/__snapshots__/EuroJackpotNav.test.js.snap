// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <EuroJackpotNav />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <ul>
        <NavLink
                activeClassName="active"
                ariaCurrent="true"
                exact={true}
                to="/"
        >
                <li>
                        EuroJackpot
                </li>
        </NavLink>
        <NavLink
                activeClassName="active"
                ariaCurrent="true"
                to="/results-winning-numbers"
        >
                <li>
                        EuroJackpot Results & Price Breakdown
                </li>
        </NavLink>
        <NavLink
                activeClassName="active"
                ariaCurrent="true"
                to="/help"
        >
                <li>
                        EuroJackpot Help & Faq
                </li>
        </NavLink>
</ul>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            exact={true}
            to="/"
>
            <li>
                        EuroJackpot
            </li>
</NavLink>,
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            to="/results-winning-numbers"
>
            <li>
                        EuroJackpot Results & Price Breakdown
            </li>
</NavLink>,
          <NavLink
            activeClassName="active"
            ariaCurrent="true"
            to="/help"
>
            <li>
                        EuroJackpot Help & Faq
            </li>
</NavLink>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": <li>
              EuroJackpot
</li>,
            "exact": true,
            "to": "/",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "EuroJackpot",
            },
            "ref": null,
            "rendered": "EuroJackpot",
            "type": "li",
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": <li>
              EuroJackpot Results & Price Breakdown
</li>,
            "to": "/results-winning-numbers",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "EuroJackpot Results & Price Breakdown",
            },
            "ref": null,
            "rendered": "EuroJackpot Results & Price Breakdown",
            "type": "li",
          },
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "activeClassName": "active",
            "ariaCurrent": "true",
            "children": <li>
              EuroJackpot Help & Faq
</li>,
            "to": "/help",
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "EuroJackpot Help & Faq",
            },
            "ref": null,
            "rendered": "EuroJackpot Help & Faq",
            "type": "li",
          },
          "type": [Function],
        },
      ],
      "type": "ul",
    },
    "type": "nav",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <ul>
          <NavLink
                    activeClassName="active"
                    ariaCurrent="true"
                    exact={true}
                    to="/"
          >
                    <li>
                              EuroJackpot
                    </li>
          </NavLink>
          <NavLink
                    activeClassName="active"
                    ariaCurrent="true"
                    to="/results-winning-numbers"
          >
                    <li>
                              EuroJackpot Results & Price Breakdown
                    </li>
          </NavLink>
          <NavLink
                    activeClassName="active"
                    ariaCurrent="true"
                    to="/help"
          >
                    <li>
                              EuroJackpot Help & Faq
                    </li>
          </NavLink>
</ul>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <NavLink
              activeClassName="active"
              ariaCurrent="true"
              exact={true}
              to="/"
>
              <li>
                            EuroJackpot
              </li>
</NavLink>,
            <NavLink
              activeClassName="active"
              ariaCurrent="true"
              to="/results-winning-numbers"
>
              <li>
                            EuroJackpot Results & Price Breakdown
              </li>
</NavLink>,
            <NavLink
              activeClassName="active"
              ariaCurrent="true"
              to="/help"
>
              <li>
                            EuroJackpot Help & Faq
              </li>
</NavLink>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "ariaCurrent": "true",
              "children": <li>
                EuroJackpot
</li>,
              "exact": true,
              "to": "/",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "EuroJackpot",
              },
              "ref": null,
              "rendered": "EuroJackpot",
              "type": "li",
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "ariaCurrent": "true",
              "children": <li>
                EuroJackpot Results & Price Breakdown
</li>,
              "to": "/results-winning-numbers",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "EuroJackpot Results & Price Breakdown",
              },
              "ref": null,
              "rendered": "EuroJackpot Results & Price Breakdown",
              "type": "li",
            },
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "activeClassName": "active",
              "ariaCurrent": "true",
              "children": <li>
                EuroJackpot Help & Faq
</li>,
              "to": "/help",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "EuroJackpot Help & Faq",
              },
              "ref": null,
              "rendered": "EuroJackpot Help & Faq",
              "type": "li",
            },
            "type": [Function],
          },
        ],
        "type": "ul",
      },
      "type": "nav",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
